src/Argu/Utils.fs

```diff
@@ -152,7 +172,7 @@ type UnionCaseInfo with
 
         match uci.GetCustomAttributes typeof<'T> with
         | [||] when includeDeclaringTypeAttrs ->
-            match uci.DeclaringType.GetCustomAttributes(typeof<'T>, false) with
+            match uci.DeclaringType.GetCustomAttributes(typeof<'T>, false) |> Seq.toArray with
```

Comment:
It's good to see the CoreCLR/CoreFx teams taking this opportunity to fix up some poor APIs in the original BCL.